modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/index/hash/HashIndexTree.java

```diff
@@ -61,14 +66,21 @@ public HashIndexTree(
             AtomicLong globalRmvId,
             long metaPageId,
             @Nullable ReuseList reuseList,
+            int binaryTupleInlineSize,
```

Comment:
Can this simply be an `inlineSize`? No need to mention tuples

Reply:
Fix it.